.yamllint

```diff
@@ -8,9 +8,9 @@ rules:
   # this behavior.
   comments-indentation: disable
 
+  # Enforcing this rule would be complicated for auto-generated data right now.
+  line-length: disable
+
```

Comment:
What happened to https://github.com/cisagov/log4j-affected-db/pull/529/commits/21a68f461cf04ee5009c22e6311afdfdeff6f0fe and https://github.com/cisagov/log4j-affected-db/pull/529/commits/3547c4f28d98a0e08582f5cf02e53d102a9e5dac?

Reply:
Ah, thanks- I wasn't thinking. 👍